source/content/guides/drupal-9-migration/05-drupal-9-to-pantheon.md

```diff
@@ -0,0 +1,377 @@
+categories: [develop]
+cms: drupal-9
+tags: [code, launch, migrate, site, updates]
+reviewed: "2021-11-12"
+layout: guide
+showtoc: true
+permalink: docs/guides/drupal-9-migration/drupal-9-to-pantheon
+anchorid: drupal-9-migration/drupal-9-to-pantheon
+editpath: drupal-9-migration/05-drupal-9-to-pantheon.md
+---
+
+This doc shows how to migrate an existing Composer-managed Drupal 9 site from another platform to a new Drupal 9 site with [Integrated Composer](/integrated-composer) on Pantheon.
+
+## Overview
+
+In this migration you'll do the following:
+
+- Using Git, clone your existing site locally
+- Create a new Drupal 9 development site on Pantheon, and clone it locally
+
+Working locally across both the existing and new site codebases, you'll:
+
+- Copy selected portions of the existing site code into the new site code
+- Copy portions of `composer.json` from the existing site to the new site
+- Copy and commit the existing site's custom modules, themes, profiles, and other dependencies to the new site
+- Make any needed customizations to `settings.php` on the new site
+- Carry over any additional code customizations from the existing site to the new Pantheon Drupal 9 site code
+
+<Alert title="Integrated Composer and existing files" type="danger">
+
+Integrated Composer will break if `composer install` tries to modify any files that are committed in Git. To avoid this, [DO THIS THING *****]
+
+</Alert>
+
+## Important Notes
+
+- Integrated Composer sites require a [nested docroot](/nested-docroot) architecture. When copying over code from the existing site, be sure to retain the new site's nested docroot structure and `web` docroot name.
+  - Address the question of Composer packages being committed to version control, and how we need to not do that for IC
+- Partial `drupal-9/prepare-local-environment.md` says to install the terminus site clone plugin, but I'm not sure this is used?
+  - Edward to adjust to alias or `former-platform`
+- `composer show` - not using this
+- have them remove packages from their list that we already include (drupal/core-recommended, drupal/core-composer-scaffold..)? wikimedia merge plugin?
+
+## Will This Guide Work for Your Site?
+
+Before you continue, confirm that your site meets the following requirements:
+
+- The existing Drupal 9 site uses Composer to manage site dependencies
+- Composer-managed
+- Able to get a local copy of the existing site / access to a Git repository of it?
+
+## Create a New Drupal 9 Site
+
+1. Log in to your Pantheon account. If you don't have an account yet, [create one first](https://pantheon.io/register?docs) and familiarize yourself with the [User Dashboard](/guides/quickstart/user-dashboard) before you create a new site.
+
+1. Set up [SSH Keys](/ssh-keys) on your local computer and Pantheon account.
+
+1. Navigate to your User Dashboard and click the **Create New Site** button:
+
+  ...
```

Comment:
is this placeholder still valid?

Reply:
I think this might have been for inserting a screenshot.